marklogic-data-hub-central/ui/src/components/entities/mapping/source-entity-map/source-to-entity-map.tsx

```diff
@@ -954,20 +1072,23 @@ const SourceToEntityMap = (props) => {
                             </div>
                             :
                             <div id="dataPresent">
-                                <div className={styles.navigationCollapseButtons}>{navigationButtons}</div>
+                                
+                                <div className={styles.navigationCollapseButtons}><span><Button data-testid="expandCollapseBtn-rowKey" onClick={() => handleExpandCollapse('rowKey')} className={styles.expandCollapseBtn}>{expandedSourceFlag ? 'Collapse All' : 'Expand All'}</Button></span><span>{navigationButtons}</span></div>
```

Comment:
Can we update the testid to say expandCollapseBtn-source and expandCollapseBtn-entity so its clear than rowKey and key.

Reply:
Done